src/SFA.DAS.EmployerDemand.UnitTests/Domain/Models/WhenConstructingProviderIsInterestedEmail.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System.Collections.Generic;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.EmployerDemand.Domain.Models;
+
+namespace SFA.DAS.EmployerDemand.UnitTests.Domain.Models
+{
+    public class WhenConstructingProviderIsInterestedEmail
+    {
+        [Test, AutoData]
+        public void Then_Values_Are_Set_Correctly(
+            string recipientEmail, 
+            string recipientName,
+            int standardId,
+            string standardName, 
+            int standardLevel, 
+            string location, 
+            int numberOfApprentices,
+            int ukprn,
+            string providerName,
+            string providerEmail,
+            string providerPhone,
+            string providerWebsite)
+        {
+            var expectedTokens = new Dictionary<string, string>
+            {
+                {"AEDEmployerName", recipientName },
+                {"AEDApprenticeshipTrainingCourse", $"{standardName} (level {standardLevel})" },
+                {"AEDApprenticeshipLocation", location },
+                {"AEDNumberOfApprentices", numberOfApprentices.ToString() },
```

Comment:
One to check - but is there the similar logic for the number of apprentices? as in if zero then dont show it?

Reply:
Doh, definitely meant to do that.